docs/changelog/release-9.md

```diff
@@ -0,0 +1,78 @@
+# Darklang Release 9
+
+January 1, 2023
+
+## Language and Standard Library
+
+- Support short-circuiting `||` and `&&` expressions (initially behind a flag -
+  go to Settings to enable)
+  ([#4640](https://github.com/darklang/dark/pull/4640))
+
+- Add `List::partition_v0` function to split a list into 2 lists
+  ([#4618](https://github.com/darklang/dark/pull/4618))
+
+- Fix lambda argument names for `List::all_v0`
+  ([#4626](https://github.com/darklang/dark/pull/4626))
+
+- Add functions that operate on characters: `Char::isUppercase_v0`,
+  `Char::isLowercase_v0`, `Char::toUppercase_v1`, `Char::toLowercase_v1`,
+  `Char::isDigit_v0`, `Char::isASCII_v0`, and `Char::isASCIILetter_v0`
+  ([#4628](https://github.com/darklang/dark/pull/4628))
```

Comment:
TODO: we should update some docs corresponding to these new fns?

Reply:
Currently, `languagedetail.md` claims simply:
> In the future, Dark will support individual characters.

I think we're slightly beyond that.

Reply:
https://github.com/darklang/docs/issues/310 extracted into a separate Issue

Reply:
No harm doing this now I think.

Maybe let's experiment with fixing up docs as part of the "Release" process, see if it's easy or helpful or whatever.

Reply:
I added some details to the Chars docs. I'd appreciate a second look there

Reply:
Updated slightly.